packages/providers/fal/image-to-video/fal_image_to_video/utils/file_utils.py

```
@@ -0,0 +1,94 @@
+        model_key: Model identifier for filename
+        filename: Optional custom filename
+
+    Returns:
+        Local path of the downloaded video or None if failed
+    """
+    try:
+        if filename is None:
+            timestamp = int(time.time())
+            filename = f"{model_key}_video_{timestamp}.mp4"
+
+        print(f"📥 Downloading video...")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)